packages/slice-machine/src/modules/customType/mockHelpers.ts

```diff
@@ -1,9 +1,13 @@
 import { CustomTypeMockConfig } from "@lib/models/common/MockConfig";
-import Actions from "./";
+import {
+  deleteFieldMockConfigCreator,
+  updateFieldMockConfigCreator,
+} from "./actions";
+import { Dispatch } from "redux";
 
 export function updateWidgetGroupMockConfig(
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  dispatch: ({ type, payload }: { type: string; payload?: any }) => void
+  dispatch: Dispatch<any>
```

Comment:
still a few any types

Reply:
Here I cannot give a type to it because we don't have a type grouping all the actions in the App and as this function will be replace when working on the mock config , doing a type for that sounds a little bit overkill I think. 

What do you want to do ?

Reply:
Do we know what the payload will be?

or we could use a very permissive type like.
```
type payload = undefined | boolean | number | string | null | Array<payload> | Record<string, payload> 
```

Reply:
Fixed